DICOMPlugins/DICOMTID1500Plugin.py

```diff
@@ -124,6 +148,114 @@ def createLoadableAndAddReferences(self, datasets):
         segLoadables = segPlugin.examine([slicer.dicomDatabase.filesForSeries(segSeriesInstanceUID)])
         for segLoadable in segLoadables:
           loadable.referencedInstanceUIDs += segLoadable.referencedInstanceUIDs
+      
+      ### Additions for handling other SRs ### 
+          
+      # First, since the point/bbox/line SRs have no SEG associated, and therefore no referencedInstanceUIDs were added, we do a special case 
+      # We add to the loadable.referencedInstanceUIDs list 
+      # This only adds the particular SOPs that the bbox/lines are on. 
+      # if (loadable.ReferencedSegmentationInstanceUIDs[uid] == []):
+      #   loadable.referencedInstanceUIDs += loadable.ReferencedOtherInstanceUIDs
+      # We need to add all of the SOPs for a series 
+      if (loadable.ReferencedSegmentationInstanceUIDs[uid] == []): 
+        # Iterate through the loadable.ReferencedOtherInstanceUIDs, and get the associated SeriesInstanceUIDs 
+        SeriesInstanceUIDs = [] 
+        for SOPInstanceUID in loadable.ReferencedOtherInstanceUIDs: 
+          filePath = slicer.dicomDatabase.fileForInstance(SOPInstanceUID)
+          SeriesInstanceUIDs.append(slicer.dicomDatabase.fileValue(filePath, "0020,000E"))
+        # Then get a unique list of the SeriesInstanceUID, likely is 1
+        SeriesInstanceUIDs = list(set(SeriesInstanceUIDs))
+        # Now for each of these SeriesInstanceUIDs, get the list of SOPInstanceUIDs 
+        if (SeriesInstanceUIDs):
+          db = slicer.dicomDatabase
+          SOPInstanceUIDs = [] 
+          for SeriesInstanceUID in SeriesInstanceUIDs: 
+            fileList = db.filesForSeries(SeriesInstanceUID) 
+            for file in fileList: 
+              SOPInstanceUIDs.append(db.fileValue(file, "0008,0018"))
```

Comment:
If I understood this correctly, your strategy is to take all `SeriesInstanceUID` values, get all instances for those series and add them as references, but often you will see specific instances referenced that are part of the referenced series directly in the SR, so this may lead to duplicate inclusion of referenced instances.

Reply:
Hmm I do not think I will have duplicate instances, as I make sure to keep only the unique SeriesInstanceUIDs. 

Fixed in this [branch](https://github.com/deepakri201/QuantitativeReporting/tree/add_planar_annot).